docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -141,6 +141,94 @@ which groups these values were observed, as well as the difference and ratio
+result of failing to consider the way in with social context and technology
+are interrelated, as well as a deeper understanding of "the social" in order to
+solve problems Ackerman [#5]_.
+
+The Framing Trap
+^^^^^^^^^^^^^^^^
+
+Selbst et al. [#4]_ define this as a "failure to model the entire system over
+which a social criterion, such as fairness, will be enforced".
+
+It is the practice of failing to consider the full picture surrounding sources of
+bias.
+
+Labelling is a risk assessment setting. Risk assessment may be used as a predictive
+model whose goal is to assess the risk of an individual to ease the decision-making
+process. For example, using risk assessment to decide whether a person is eligible
+for mortgage approval. Another example is using risk assessment to decide whether a
+person is likely to re-engage in criminal behaviour as they move through the criminal
+justice pipeline.
+
+As far as the outcomes of these decisions go, in the first scenario an individual may be
+approved or not to get a mortgage which then impacts their ability to have housing security,
+gain wealth and social status. In the second scenario an individual may be released pretrial,
+or bail may be required. This assessment may also include sentencing and parole hearing.
+Risk assessment tools may be used as recommendations towards an outcome. However there is no
+guarantee that they will be used consistently and thus bias is introduced. For example, a
+party responsible for choosing the outcome decides to ignore the recommendation as a result
+of personal bias, even when the model takes into account demographic data. In this example
+a model that fails to take into account bias present in those judging culminates in a
+biased result.
+
+The Portability Trap
+^^^^^^^^^^^^^^^^^^^^
+
+Selbst et al. [#4]_ define this as a "failure to understand how repurposing algorithmic
+solutions designed for one social context may be misleading, inaccurate, or otherwise
+do harm when applied to a different context".
+
+It is the practice of reusing others work without taking into account lack of
+transferability to a given social situation, resulting in harmful consequences.
+
+The Formalism Trap
+^^^^^^^^^^^^^^^^^^
+
+Selbst et al. [#4]_ define this as a "failure to account for the full meaning of social concepts
+such as fairness, which can be procedural, contextual, and contestable, and cannot be resolved
+through mathematical formalisms".
+
+It is the practice of implementing mathematical and statistical models, along with
+corresponding assumptions, that fail to take into consideration the social, demographic,
+economic, or otherwise non-technical aspect that make up the phenomenon being studied.
+
+The Ripple Effect Trap
+^^^^^^^^^^^^^^^^^^^^^^
+
+Selbst et al. [#4]_ define this as a "failure to understand how the insertion of technology into
+an existing social system changes the behaviors and embedded values of the pre-existing system".
+
```

Comment:
This one is a bit more straightforward than the formalism trap, but a somewhat more extended definition may be helpful! (e.g., something that gets at the fact that ML systems may change _people's_ behaviors, organizations' incentives, or other social forces or actors).

And, as with the previous two, an example would be super helpful here!

Reply:
Added similar example, but substituted for loans as I mention the judge one in the next section